libs/sdk-bindings/src/uniffi_binding.rs

```diff
@@ -195,7 +196,7 @@ impl BlockingBreezServices {
         rt().block_on(self.breez_services.report_issue(req))
     }
 
-    pub fn sweep(&self, req: SweepRequest) -> SdkResult<SweepResponse> {
+    pub fn sweep(&self, req: RedeemOnchainFundsRequest) -> SdkResult<RedeemOnchainFundsResponse> {
         rt().block_on(self.breez_services.sweep(req))
     }
```

Comment:
```suggestion
    pub fn redeem_onchain_funds(&self, req: RedeemOnchainFundsRequest) -> SdkResult<RedeemOnchainFundsResponse> {
        rt().block_on(self.breez_services.redeem_onchain_funds(req))
    }
```

Reply:
Oh, my bad. Thanks, fixed with the latest push.